minecraft/networking/types.py

```diff
@@ -4,6 +4,7 @@
 """
 import struct
 import uuid
+from collections import namedtuple
 
 
 class Type(object):
```

Comment:
Oh, and I almost forgot: the class `Type` here needs to also have `__slots__ = ()`, otherwise every instance of `Position` will have an unnecessary empty `__dict__` allocated.